kura/org.eclipse.kura.localization.resources/src/main/java/org/eclipse/kura/localization/resources/WireMessages.java

```diff
@@ -704,8 +704,10 @@
             + "If set to true, the values for all channels will be read and emitted, if set to false, only the value for the channel related to the event will be emitted.")
     public String emitAllChannelsDescription();
 
-    @En("If set to false, a timestamp per channel will be included in the emitted envelope, if set to true, only a single timestamp will be emitted.")
-    public String singleTimestampDescription();
+    @En("If set to PER_CHANNEL, the component will emit a driver-generated timestamp per channel property."
+            + " If set to SINGLE_ASSET_GENERATED, the component will emit a single timestamp per request, generated by the Asset itself before emitting the envelope."
+            + " If set to SIGNLE_DRIVER_GENERATED_MAX or SIGNLE_DRIVER_GENERATED_MIN, the component will emit a single driver generated timestamp being respectively the max (most recent) or min (oldest) among the timestamps of the channels.")
```

Comment:
review the spelling of SIGNLE_DRIVER_GENERATED_MAX and SIGNLE_DRIVER_GENERATED_MIN